docs/how_to_use/how_to_migrate_your_circuit.md

```diff
@@ -0,0 +1,132 @@
+
+# How to migrate you circuit in Rust
+
+Halo2 is currently the most widely used Proving System, with implementations such as [zcash/halo2](https://github.com/zcash/halo2), [privacy-scaling-explorations/halo2](https://github.com/privacy-scaling-explorations/halo2), and [scroll-tech/halo2](https://github.com/scroll-tech/halo2).
+Proving programs equipped with a Halo2 can experience a decrease in proof generation costs when switching the backend to Tachyon.
+
+## Requirements
+
+1. Replace `halo2` library in your program with [tachyon-halo2](https://github.com/kroma-network/halo2).
+2. Install the tachyon debian package, see *Debian packaging* section in [how_to_build.md](./how_to_build.md)
+
+## Your Circuit
+
+Let's assume that your circuit has an interface almost identical to the [example circuit](../../vendors/halo2/src/circuits/shuffle_circuit.rs) we have prepared in advance.
```

Comment:
Do we need such assumption?

Reply:
Maybe we need to mention that we only provide with `SHPlonk` and `GWCProver` with `bn254`.